src/Bundle/test/src/Subscription/Entity/Subscription.php

```diff
@@ -47,6 +48,8 @@
     alias: 'app.subscription',
     section: 'ajax',
     routePrefix: '/ajax',
+    normalizationContext: ['groups' => 'subscription:read'],
```

Comment:
A small thing that we could improve here is the usage of PHP consts as groups. Not needed, but recently I started to think about it as the default approach and wanted to share this thought :)